core/remotes/docker/fetcher.go

```diff
@@ -478,6 +478,10 @@ func (r dockerFetcher) open(ctx context.Context, req *request, mediatype string,
 	})
 
 	remaining, _ := strconv.ParseInt(resp.Header.Get("Content-Length"), 10, 0)
+	if remaining <= minChunkSize {
```

Comment:
`chunkSize` instead?

Reply:
Oh, yes, that makes more sense ! Updated & Rebased.